hrms/hr/doctype/leave_application/leave_application.py

```diff
@@ -953,7 +953,7 @@ def get_leave_allocation_records(employee, date, leave_type=None):
 			& (Ledger.docstatus == 1)
 			& (Ledger.transaction_type == "Leave Allocation")
 			& (Ledger.employee == employee)
-			& (Ledger.is_expired == 0)
+			& (LeaveAllocation.expired == 0)
```

Comment:
Ledger getting expired and allocation getting expired is the same right? How does the earlier one not work?

Reply:
Expired leaves were still shown despite the ledger filter applied